packages/webview_flutter/webview_flutter_platform_interface/lib/v4/src/types/platform_webview_widget_creation_params.dart

```diff
@@ -21,26 +22,26 @@ import '../platform_webview_controller.dart';
 /// changes.
 ///
 /// ```dart
-/// class WKWebViewWidgetCreationParams extends PlatformWebViewWidgetCreationParams {
-///   WKWebViewWidgetCreationParams._(
-///     // This parameter prevents breaking changes later.
-///     // ignore: avoid_unused_constructor_parameters
+/// class AndroidWebViewWidgetCreationParams
+///     extends PlatformWebViewWidgetCreationParams {
+///   AndroidWebViewWidgetCreationParams({
+///     super.key,
+///     super.layoutDirection,
+///     super.gestureRecognizers,
+///     this.platformSpecificFieldExample,
```

Comment:
Copypasta?

Reply:
I don't understand you question. Do you think this should be a better name?

Reply:
Sorry, I somehow missed that this was a change to the comment showing use. I thought this was actual code and that you'd accidentally included the parameter from the example.